package.json

```diff
@@ -61,7 +62,8 @@
   },
   "husky": {
     "hooks": {
-      "pre-commit": "lint-staged"
+      "pre-commit": "lint-staged",
+      "prepare-commit-msg": "exec < /dev/tty && git cz --hook"
```

Comment:
I don't think this will work on windows or other environments.
Also: I'm not entirely sure that it will be a good idea to force everyone to use `git-cz`, because some times it is just easier to write your commit messages in vi/vim or whatever your preferred commit message editor is.

Maybe we could just have the `commitizen` config in this repository and then also use https://github.com/conventional-changelog/commitlint to reject any invalid messages in case the committer is not using `git-cz`?

This is similar to [how we do it in `untool`](https://github.com/untool/untool/blob/master/package.json#L134-L161)